.github/dependabot.yml

```diff
@@ -1,6 +1,11 @@
 version: 2
 updates:
-- package-ecosystem: github-actions
-  directory: /
-  schedule:
-    interval: weekly
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: github-actions
+    directory: /.github/actions/setup-nix
+    schedule:
+      interval: weekly
```

Comment:
After checking some dependabot issues, I modified the file according to this example: https://github.com/elastic/apm-pipeline-library/blob/main/.github/dependabot.yml#L22-L29